compiler/plc_driver/src/lib.rs

```diff
@@ -247,6 +247,11 @@ pub fn compile_with_options(compile_options: CompilationContext) -> Result<()> {
         .index(ctxt.provider())
         .annotate(ctxt.provider());
 
+    if compile_parameters.output_ast {
+        println!("{:?}", annotated_project.units);
```

Comment:
Although `:?` prints a debug version, it's hard to read when the output contains many elements. Fortunately Rust can ["pretty print"](https://doc.rust-lang.org/std/fmt/trait.Debug.html#examples) by replacing `:?` with `:#?`, making the output more readable.
```suggestion
        println!("{:#?}", annotated_project.units);
```

Reply:
Ah, wasn't aware. That looks way better!